### DISCUSSION TITLE
Need help finding a summary output file

### DISCUSSION THREAD

**Discussion by rosspdu** [hahnlab/CAFE5#175] [Q&A] [answered]
Hello,

I am trying to find if there's any summary file generated from the cafe5 run I did previously but I couldn't find any. I wanted to find out the overall dynamic of gene family evolution (expansion and contraction) across the phylogeny of my study system. It appears that the global pattern is expansion but I wanted to find out if cafe5 produces an output summary showing a significant value to support this.

Thank you,
Ross

**Comment by benfulton** [comment, marked as answer]
Just for completeness, this was answered on the mailing list:

"CAFE does give a branch by branch accounting of all gains and losses, but no single number. You could sum together all gains and losses if you are looking for some sort of global tally."
